desktop/core/src/desktop/api2.py

```diff
@@ -860,7 +864,7 @@ def is_reserved_directory(doc):
   stdout = string_io()
   try:
     with transaction.atomic(): # We wrap both commands to commit loaddata & sync
-      management.call_command('loaddata', f.name, verbosity=3, traceback=True, stdout=stdout, commit=False) # We need to use commit=False because commit=True will close the connection and make Document.objects.sync fail.
+      management.call_command('loaddata', f.name, verbosity=3, traceback=True, stdout=stdout)
```

Comment:
Is there an equivalent for doing commit=False?

Reply:
https://github.com/jazzband/django-nose/pull/308 This PR is related to this. I think this the right way of doing this. @romainr

Reply:
:+1:

Reply:
tracking @ https://github.com/cloudera/hue/pull/1652